src/test/java/seedu/address/testutil/TypicalAppObjects.java

```diff
@@ -215,7 +215,7 @@ public static AppointmentSchedule getEmptyAppointmentSchedule() {
     }
 
     public static List<Doctor> getTypicalDoctors() {
-        return new ArrayList<>(Arrays.asList(DR_GREY, DR_WHO, DR_STRANGE, DR_JEKYLL, DR_MURPHY, DR_DRAKE));
+        return new ArrayList<>(Arrays.asList(DR_GREY, DR_WHO, DR_STRANGE, DR_JEKYLL));
```

Comment:
add your own extra doctors, doing this may ruin others' tests since typicalappointments make use of these 2 doctors u removed...

follow the patient convention (lines 88 to 102) and write 2 extra doctors. use a UUID version 4 generator off the web if u need a UUID.